core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -2011,6 +2015,34 @@ public ImportDestinationArguments importDirectory(String directory) {
     return new BulkImport(directory, context);
   }
 
+  @Override
+  public TimeType getTimeType(final String tableName) throws TableNotFoundException {
+    if (tableName.equals(RootTable.NAME)) {
+      throw new IllegalArgumentException("accumulo.root table has no TimeType");
+    }
+    String systemTableToCheck =
+        MetadataTable.NAME.equals(tableName) ? RootTable.NAME : MetadataTable.NAME;
+    final Scanner scanner = context.createScanner(systemTableToCheck, Authorizations.EMPTY);
```

Comment:
suggest moving this line to right before scanner is used and put in try-with-resources block.

Reply:
With the changes to be made with Keith's suggestion below this change will no longer be necessary.